lib/Cake/TestSuite/CakeTestCase.php

```diff
@@ -164,6 +166,8 @@ public function tearDown() {
 		if (isset($_GET['debug']) && $_GET['debug']) {
 			ob_flush();
 		}
+		unset($this->_configure);
+		unset($this->_pathRestore);
```

Comment:
You can do all of this in one unset call like `unset($this->_configure, $this->_pathRestore)`.